src/utilities/TwilioFunctions.js

```diff
@@ -1,15 +1,11 @@
 const axios = require('axios')
 
-const accountSID = process.env.REACT_APP_TWILIO_SID
-const twilioToken = process.env.REACT_APP_TWILIO_AUTH_TOKEN
-const messagingWebhook = process.env.REACT_APP_TWILIO_BRAVE_WEBHOOK_URL
-const allSensorMessagesSID = process.env.REACT_APP_TWILIO_ALL_SENSOR_MESSAGES_SID
```

Comment:
Removed these because it didn't look like they were being used in this file at all.